EIPS/eip-3076.md

```diff
@@ -0,0 +1,292 @@
+---
+eip: 3076
+title: Validator client interchange format (slashing protection)
+author: Michael Sproul (@michaelsproul), Sacha Saint-Leger (@sachayves), Danny Ryan (@djrtwo)
+discussions-to: https://discord.com/channels/595666850260713488/598292067260825641/
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-27
+---
+
+## Simple Summary
+
+A JSON interchange format for Serenity (eth2) that contains the necessary slashing protection information required to safely migrate keys between clients.
+
+## Abstract
+
+ A standard format for transferring a key's signing history allows validators to easily switch between clients without the risk of signing conflicting messages. While a [common keystore format](https://eips.ethereum.org/EIPS/eip-2335) provides part of the solution, it does not contain any information about a key's signing history. For a validator moving their keys from client A to client B, this can lead to scenarios in which client B inadvertently signs a message that conflicts with an earlier message signed with client A.  
+ 
+ We propose a database interchange format that helps solve this problem. The format contains a record of all blocks and attestations signed by a set of validators. 
```

Comment:
This content currently feels much more like motivation rather than an abstract.  Ideally, the abstract is a human readable and terse version of the specification.  A couple paragraphs that gives the reader the highlights of the specification without getting bogged down in the edge cases and minutia.  Someone may read the abstract to try to "get the gist" of the specification and decide whether this is an EIP that they want to implement.